kolibri/plugins/learn/assets/src/views/ExamPage/index.vue

```diff
@@ -64,7 +64,7 @@
               :answerState="currentAttempt.answer"
               @interaction="saveAnswer"
             />
-            <MissingResourceAlert v-else :multiple="false" />
+            <ResourceSyncingUiAlert v-else :multiple="false" />
```

Comment:
The `multiple` prop is not defined on `ResourceSyncingUiAlert` - needs to be added and passed through.